views/personal/maritalStatus-edit.pug

```diff
@@ -0,0 +1,45 @@
+extends ../base
+
+block variables
+  -var title = 'Change your Marital Status'
+
+block content
+
+  h1 #{title}
+
+  div
+    p Please choose the correct status from the options listed below.
+
+    checked=(data && data.personal.maritalStatus.toLowerCase()=='married')
+
+
+  form.benefits-form(method='post')
+    div
+        fieldset.fieldset
+            legend.fieldset__legend 
+                p Marital Status
+            .radios
+                .radios__item
+                    input.radios__input(id="maritalStatusMarried" name="maritalStatus" type="radio" value="Married" checked=(data && data.personal.maritalStatus.toLowerCase()=='married' || !data))
+                    label.radios__label(for="maritalStatusMarried") Married or common-law
+                .radios__item
+                    input.radios__input(id="maritalStatusWidowed" name="maritalStatus" type="radio" value="Widowed" checked=(data && data.personal.maritalStatus.toLowerCase()=='widowed'))
+                    label.radios__label(for="maritalStatusWidowed") Widowed
+                .radios__item
+                    input.radios__input(id="maritalStatusDivorced" name="maritalStatus" type="radio" value="Divorced" checked=(data && data.personal.maritalStatus.toLowerCase()=='divorced'))
+                    label.radios__label(for="maritalStatusDivorced") Divorced
+                .radios__item
+                    input.radios__input(id="maritalStatusSeparated" name="maritalStatus" type="radio" value="Separated" checked=(data && data.personal.maritalStatus.toLowerCase()=='separated'))
+                    label.radios__label(for="maritalStatusSeparated") Separated
+                .radios__item
+                    input.radios__input(id="maritalStatusSingle" name="maritalStatus" type="radio" value="Single" checked=(data && data.personal.maritalStatus.toLowerCase()=='single'))
+                    label.radios__label(for="maritalStatusSingle") Single
```

Comment:
Maybe eventually we should make these an `/includes` as well, but for now this looks good 👍